corehq/messaging/scheduling/forms.py

```diff
@@ -2816,6 +2847,18 @@ class ConditionalAlertScheduleForm(ScheduleForm):
         required=False,
     )
 
+    sms_stale_after = IntegerField(
+        label=ugettext_lazy("Make SMS stale after (days)"),
+        required=False,
+        min_value=MIN_SMS_STALE_INTERVAL,
+        max_value=MAX_SMS_STALE_INTERVAL,
+        help_text=ugettext_lazy("SMS is marked as stale in 7*24 hours by default.\
```

Comment:
New users won't know what "stale" means. How about adding something like "Stale SMSes will not be sent." to the end?